packages/aws-rfdk/lib/deadline/lib/configure-spot-event-plugin.ts

```diff
@@ -0,0 +1,643 @@
+      DeleteInterruptedSlaves: props.configuration?.deleteEC2SpotInterruptedWorkers ?? false,
+      DeleteTerminatedSlaves: props.configuration?.deleteSEPTerminatedWorkers ?? false,
+      IdleShutdown: props.configuration?.idleShutdown?.toMinutes({integral: true}) ?? 10,
+      Logging: props.configuration?.loggingLevel ?? SpotEventPluginLoggingLevel.STANDARD,
+      PreJobTaskMode: props.configuration?.preJobTaskMode ?? SpotEventPluginPreJobTaskMode.CONSERVATIVE,
+      Region: props.configuration?.region ?? region,
+      ResourceTracker: props.configuration?.enableResourceTracker ?? true,
+      StaggerInstances: props.configuration?.maximumInstancesStartedPerCycle ?? 50,
+      State: props.configuration?.state ?? SpotEventPluginState.GLOBAL_ENABLED,
+      StrictHardCap: props.configuration?.strictHardCap ?? false,
+    };
+    const spotFleetRequestConfigs = this.mergeSpotFleetRequestConfigs(props.spotFleets);
+
+    const properties: SEPConfiguratorResourceProps = {
+      connection: {
+        hostname: props.renderQueue.endpoint.hostname,
+        port: props.renderQueue.endpoint.portAsString(),
+        protocol: props.renderQueue.endpoint.applicationProtocol,
+        caCertificateArn: props.renderQueue.certChain?.secretArn,
+      },
+      spotFleetRequestConfigurations: spotFleetRequestConfigs,
+      spotPluginConfigurations: pluginConfig,
+    };
+
+    const resource = new CustomResource(this, 'Default', {
+      serviceToken: configurator.functionArn,
+      resourceType: 'Custom::RFDK_ConfigureSpotEventPlugin',
+      properties,
+    });
+
+    // Prevents a race during a stack-update.
+    resource.node.addDependency(configurator.role!);
+
+    // We need to add this dependency to avoid failures while deleting a Custom Resource:
+    // 'Custom Resource failed to stabilize in expected time. If you are using the Python cfn-response module,
+    // you may need to update your Lambda function code so that CloudFormation can attach the updated version.'.
+    // This happens, because Route Table Associations are deleted before the Custom Resource and we
+    // don't get a response from 'doDelete()'.
+    // Ideally, we would only want to add dependency on 'internetConnectivityEstablished' as shown below,
+    // but it seems that CDK misses dependencies on Route Table Associations in that case:
+    // const { internetConnectivityEstablished } = props.vpc.selectSubnets(props.vpcSubnets);
+    // resource.node.addDependency(internetConnectivityEstablished);
+    resource.node.addDependency(props.vpc);
+
+    // /* istanbul ignore next */
+    // Add a dependency on the render queue to ensure that
+    // it is running before we try to send requests to it.
+    resource.node.addDependency(props.renderQueue);
+
+    this.node.defaultChild = resource;
+  }
+
+  private tagsSpecifications(fleet: SpotEventPluginFleet, resourceType: SpotFleetResourceType): IResolvable {
```

Comment:
Maybe drop the "s" in tags?
```suggestion
  private tagSpecifications(fleet: SpotEventPluginFleet, resourceType: SpotFleetResourceType): IResolvable {
```
This could also use a short doc string.

Reply:
Removed.